doc/wiki/source_code/Version_Control_System.md

```diff
@@ -62,20 +62,9 @@ the format for github wiki pages, where the site will be migrated
 to due to end of life of the mediawiki at ANL.
 ```
 ## Git
-### Why Git?
```

Comment:
Please update the commit message. You are combining two files rather than created a new file.